src/Internal/EventStoreConnectionLogicHandler.php

```diff
@@ -344,6 +345,7 @@ public function tcpConnectionError(TcpPackageConnection $tcpPackageConnection, T
             return;
         }
 
+        /** @psalm-suppress PossiblyNullReference */
```

Comment:
I had to add a bunch of those comments in this class. The class itself is internal and there is no way that the connection is null in those cases, but psalm can't know.